charts/simple-app/templates/deployment.yaml

```diff
@@ -132,6 +132,9 @@ spec:
 
         {{- end }}
         {{- end }}
+        {{- if and .Values.datadog.enabled .Values.datadog.enableLogging .Values.datadog.service }}
+        ad.datadoghq.com/writer.logs: '[{"source":{{ .Values.datadog.service | quote}},"service":{{ .Values.datadog.service | quote}}]'
```

Comment:
The container name needs to be parameterized here - “writer” in this case is what needs to be replaced with a template.

Reply:
There are various other parameters that can be used (and we should encourage them) by the developer to parse the logs as they flow through. We should put in some of those here as parameters so that we can make it easy to do things like strip PII, skip noisy log lines, etc…

Reply:
Added source and service parameters. If not defined it will use default values (.Release.Name). Also added support for log processing rules. https://docs.datadoghq.com/agent/logs/advanced_log_collection/?tab=configurationfile